server/dash/dashboards/neighborhood_recent.py

```diff
@@ -31,6 +30,33 @@
 # DATA
 df_path = f"/requests/updated?start_date={start_date}&end_date={end_date}"
 print(" * Downloading data for dataframe")
+
+BATCH_SIZE = 10000
+
+def batch_get_data(url):
```

Comment:
I know you're just copying this from another file, but could you add a docstring for this?

Reply:
Thanks for the catch - resolved